src/MongoDB/Manager.c

```diff
@@ -79,6 +79,16 @@ static bool php_phongo_manager_merge_context_options(zval* zdriverOptions TSRMLS
 		return false;
 	}
 
+	/* When running PHP in debug mode, php_error_docref duplicates the current
+	 * scope, leading to a COW violation in zend_hash_merge and
+	 * zend_symtable_str_del (called by php_array_unsetc). This macro allows
+	 * that violation in debug mode and is a NOOP when in non-debug. */
+#ifdef HT_ALLOW_COW_VIOLATION
```

Comment:
`#if PHP_VERSION_ID >= 70200` may be more consistent with what we do elsewhere (e.g. UTCDateTime.c). This also makes it easier to grep when we bump minimum PHP versions and identify obsolete conditionals.

I don't feel strongly about this, so if you prefer the more tactical macro check I'm OK with leaving this as-is.

Reply:
You make a good point, version numbers are easier to clean up later.